template/template.go

```diff
@@ -238,6 +238,20 @@ func NewTemplateExpander(text string, name string, data interface{}, timestamp c
 				}
 				return fmt.Sprintf("%.4g%ss", v, prefix)
 			},
+			"humanizeTimestamp": func(v interface{}) string {
+				switch ts := v.(type) {
```

Comment:
I've kept the console templates to exactly one type for simplicity. I'd suggest adjusting the `query` function to return this as a float64 in seconds instead, so that all the other operations will work as expected.